File: src/main/java/no/nav/service/pensjon/regler/to/RegulerBeregningRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.ArrayList;
import java.util.Date;

import no.nav.domain.pensjon.regler.beregning.Beregning;
import no.nav.domain.pensjon.regler.grunnlag.Persongrunnlag;
import no.nav.domain.pensjon.regler.grunnlag.Uttaksgrad;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

public class RegulerBeregningRequest extends ServiceRequest {

	private Beregning beregning1967;
	private Date virkFom;
	private ArrayList<Uttaksgrad> uttaksgradListe;
	private ArrayList<VilkarsVedtak> brukersVilkarsvedtakListe;
	private Persongrunnlag sokersPersongrunnlag;
	private Persongrunnlag epsPersongrunnlag;
	

}

File: src/main/kotlin/no/nav/pensjon/regler/to/RegulerBeregningRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning.Beregning
import no.nav.pensjon.regler.domain.grunnlag.Persongrunnlag
import no.nav.pensjon.regler.domain.grunnlag.Uttaksgrad
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak
import java.util.*
import java.time.LocalDate

class RegulerBeregningRequest : ServiceRequest() {
    var beregning1967: Beregning? = null
    @Deprecated("Use virkFomLd instead")
    var virkFom: Date? = null
    var virkFomLd: LocalDate? = null
    var uttaksgradListe: ArrayList<Uttaksgrad> = ArrayList()
    var brukersVilkarsvedtakListe: ArrayList<VilkarsVedtak> = ArrayList()
    var sokersPersongrunnlag: Persongrunnlag? = null
    var epsPersongrunnlag: Persongrunnlag? = null
}